sass/_theme_rst.sass

```diff
@@ -53,8 +50,41 @@
     margin-left: $base-line-height
     line-height: $base-line-height
     margin-bottom: $base-line-height
-  .literal-block, pre.literal-block
-    @extend .codeblock
+
+  pre.literal-block, div[class^='highlight']
+    border: 1px solid $table-border-color
+    padding: 0px
+    overflow-x: auto
+    line-height: normal
+    // 1px hack otherwise border won't show. lame
+    margin: 1px 0 $base-line-height 0
+    div[class^='highlight']
+      border: none
+      margin: 0
+  // Needs 100% width for line highlighting to work properly
+  div[class^='highlight'] td.code
+    width: 100%
+  .linenodiv pre
+    border-right: solid 1px lighten($table-border-color, 2%)
+    margin: 0
+    padding: $base-line-height / 2 $base-line-height / 2
+    font-family: $code-font-family
+    font-size: 12px
+    line-height: normal
```

Comment:
Perhaps `font-size` and `line-height` should be moved to a new CSS rule for line numbering and actual code. That will prevent them from going out of sync in future.

``` sass
pre.literal-block, div[class^='highlight'] pre, .linenodiv pre
    font-size: 12px
    line-height: normal
```

Adding `pre.literal-block` to this rule also aligns the font size for literal blocks, which look out of place now:
![literal code block](https://user-images.githubusercontent.com/3284111/34204282-51926096-e57e-11e7-9bf7-e86c73329768.png)

Reply:
Done